src/suite.js

```diff
@@ -14,26 +14,33 @@ const defaultOptions = {
   minSamples: 20
 };
 
+const targetList = [
+  "acorn",
+  "babel",
+  "babylon",
+  "buble",
+  "chai",
+  "coffeescript",
+  "espree",
+  "esprima",
+  "jshint",
+  "lebab",
+  "prepack",
+  "prettier",
+  "source-map",
+  "typescript",
+  "uglify-es",
+  "uglify-js"
+];
+
 const suite = new Benchmark.Suite();
+const targetItems = ONLY ? [ONLY] : null;
```

Comment:
Why `null` here, and then `targetItems || targetList` on the next line? Why not `const targetItems = ONLY ? [ONLY] : targetList`?